### DISCUSSION TITLE
1970( Modern Web Apps 2017) - Atualização do curso

### DISCUSSION THREAD

**Discussion by jp17paulo** [balta-io/forum#372] [Tira Dúvidas] [answered]
Seria muito interessante se houvesse uma versão mais atual do curso, com tecnologias mais atuais pois o curso é bem completo.

Caso já exista outro também com as mesmas características (Criação de um app completo e publicação) e puder por gentileza indicar.

**Comment by andrebaltieri** [comment, marked as answer]
🚀
Vamos ter!

**Comment by Luhc011** [comment]
Modelando domínio ricos poderia ter uma atualização também, é nítida sua evolução didática da época da publicação para os cursos mais atuais.
